pkgs/development/libraries/libjodycode/default.nix

```diff
@@ -32,8 +35,8 @@ stdenv.mkDerivation rec {
   meta = with lib; {
```

Comment:
If you're going to fixup `finalAttrs`, you should remove this `with lib;` while you're at it.

Reply:
not a blocker, but I would very much appreciate if you go through with it :)

Reply:
Will do both.